http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -216,18 +226,38 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                   petMi,
                   storageContainer,
                   BatchAccountKey(batchAccountKey),
-                  applicationInsightsComponent.connectionString()
+                  applicationInsightsComponent.connectionString(),
+                  appChartPrefix
+                ),
+                createNamespace = true
+              )
+              .run(authContext)
+          } yield ()
+        case AppType.Wds =>
+          for {
+            _ <- helmClient
+              .installChart(
+                app.release,
+                app.chart.name,
+                app.chart.version,
+                buildWdsChartOverrideValues(
```

Comment:
We could combine this with the buildCromwellChartOverrideValues function and take appType as a param. I think this is cleaner since wds overrides requires different parameters & depends on different resource creation, so I expect these to diverge to some extent over time

Reply:
This seems fine by me :D